avs_rvtools_analyzer/templates/analyze.html

```diff
@@ -41,6 +41,69 @@ <h5 class="card-title">{{ risk_name.replace('detect_', '').replace('_', ' ').tit
     </div>
 </div>
 
+<!-- Azure OpenAI Disclaimer - Always visible when AI is available -->
+<div class="alert alert-warning ai-disclaimer" role="alert" style="display: none;">
+    <i class="fas fa-exclamation-triangle"></i> <strong>Disclaimer:</strong> 
+    AI-generated suggestions are for informational purposes only and should be thoroughly reviewed by qualified professionals. 
+    The tool maintainers cannot be held responsible for the accuracy or consequences of the provided suggestions. 
+    Always validate recommendations with your specific environment and requirements.
+</div>
```

Comment:
We will keep this message if the AI configuration is configured in the backend side. As the UI configuration will be remove it should help to simplify the logic of this page.